Makefile

```
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REGISTRY                                   := eu.gcr.io/gardener-project/gardener
+REGISTRY                                   := europe-docker.pkg.dev/gardener-project/public/gardener
```

```suggestion
REGISTRY                                   := europe-docker.pkg.dev/gardener-project/snapshots/gardener
```
This is for `make docker-images` and `make docker-push` only. These make target are used by developers only to push there local development, so it should be `snapshots`.